### PR TITLE
[1LP][RFR] New region tests

### PR DESCRIPTION
**Pull request by otsuman**
{{pytest: cfme/tests/configure/test_region.py}}

RHCF3-11856

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f02658e25e20865c889e0179cc888ae5f599a0a6:

`cfme/tests/configure/test_region.py`:
- :heavy_exclamation_mark: Line 33:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/base/ui.py [hunk 1]
```diff
@@ -796,6 +797,17 @@ def is_displayed(self):
...
 
+class RegionChangeNameView(RegionView):
+    region_description = Input("region_description")
+    save = Button('Save')
+    reset = Button('Reset')
+    cancel = Button('Cancel')
+
+    @property
+    def is_displayed(self):
+        return self.region_description.is_displayed
```
Comment (quarckster) [inline cfme/base/ui.py]: **REQUIRED**
```python
@property
def is_displayed(self):
    return self.region_description.is_displayed and super(RegionChangeNameView, self).is_displayed
```

File: cfme/base/ui.py [hunk 2]
```diff
@@ -766,6 +766,7 @@ class RegionView(ConfigurationView):
     @View.nested
     class details(Tab):  # noqa
         TAB_NAME = "Details"
+        table = Table(locator="//td[@title='Edit this Region']/../../..")
```
Comment (izapolsk) [inline cfme/base/ui.py]: REQUIRED: I'd rather change this to something like Table(locator=".//table[./tbody/tr/td[blabla]]")

File: cfme/tests/configure/test_region.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+def test_description_change(appliance, request):
+    """Test changing region description"""
+    view = navigate_to(appliance.server.zone.region, 'ChangeRegionName')
+
+    def _reset_region_description(description, view):
+        view.details.table.row().click()
+        view.region_description.fill(description)
+        view.save.click()
+    region_description = fauxfactory.gen_alphanumeric(5)
+    old_desctiption = view.region_description.read()
```
Comment (izapolsk) [inline cfme/tests/configure/test_region.py]: REQUIRED: typo

**Comment by otsuman** [comment]
@mmojzis

File: cfme/tests/configure/test_region.py (hunk 3 above)
Comment (otsuman) [reply cfme/tests/configure/test_region.py]: fixed

File: cfme/base/ui.py (hunk 2 above)
Comment (otsuman) [reply cfme/base/ui.py]: fixed

**Comment by izapolsk** [comment]
@otsuman, please let me know when PRT is done and I'll merge it.

**Pull request merged**